src/main/java/htsjdk/tribble/readers/TabixReader.java

```diff
@@ -533,6 +554,8 @@ public static void main(String[] args) {
                     System.out.println(s);
             }
```

Comment:
This main method is bad and untested.  Instead of trying to fix it can you just delete it?